build.gradle

```diff
@@ -43,12 +23,12 @@ ext {
 
 def dependencyVersions = [
         opensaml              : "$opensaml_version",
-        ida_utils             : '2.0.0-392',
+        ida_utils             : '2.0.0-395',
```

Comment:
https://repo1.maven.org/maven2/uk/gov/ida/common-utils/ is this the ida_utils which is at version 396?